packages/fluentui/accessibility/src/behaviors/Datepicker/datepickerBehavior.ts

```diff
@@ -18,4 +26,7 @@ export const datepickerBehavior: Accessibility<DatepickerBehaviorProps> = props
   },
 });
 
-export type DatepickerBehaviorProps = never;
+export type DatepickerBehaviorProps = {
+  'aria-labelledby'?: string;
+  'aria-invalid'?: boolean | 'true' | 'false';
```

Comment:
Why do we need to support the string values ?

Reply:
I was thinking and considering usage, when it is defined like attribute in "html" way:
![image](https://user-images.githubusercontent.com/22620150/103363208-d605aa00-4aba-11eb-9a87-71b412cb5f0b.png)

Reply:
Since this is really only gonna be used in the context of react, I'd rather only support the boolean. But it's your call

EDIT: letting consumers use strings might negatively impact bundling since the string would probably need to bundled, not entirely sure on this since I haven't tested this but it's a thought

Reply:
changed only for boolean :)